docs/managing-mods.md

```diff
@@ -0,0 +1,52 @@
+# Managing mods
+
+> Learn how to manage your mods using Lampray.
+
+## Prerequisites
+
+Before you start, you'll need to [build Lampray from source](./building-from-source.md).
+
+## Setting the path to Steam
```

Comment:
Update all references to reflect a standard installation (i.e. remove the `**/Build/**` references). Just leave a single callout at the top for those who built from source and where to go instead.